src/containers/urls.js

```diff
@@ -0,0 +1,7 @@
+const endPoints = {
+getCategories: 'https://ah-backend-groot.herokuapp.com/api/categories/?limit=6',
+articles: 'https://ah-backend-groot.herokuapp.com/api/articles/?limit=6',
+loginUser:'https://ah-backend-groot.herokuapp.com/api/users/login/'
+};
+
+export default endPoints;
```

Comment:
@walimike , create a base url and append the rest url string. like `${base}articles/?limit=6`